polkadot/xcm/xcm-builder/src/barriers.rs

```diff
@@ -560,7 +572,7 @@ impl DenyExecution for DenyReserveTransferToRelayChain {
 				ReserveAssetDeposited { .. }
 					if matches!(origin, Location { parents: 1, interior: Here }) =>
 				{
-					log::warn!(
+					tracing::warn!(
```

Comment:
We reserve `info, warn, error` log levels for node operators.

Runtime logs are not useful for them. For runtime logs please use `tracing` for happy-paths and `debug` for error paths.
```suggestion
					tracing::debug!(
```

```suggestion
					tracing::debug!(
```